app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    like = current_user.likes.create(post: post)
+    if like.save!
+      flash[:success] = 'like added'
+    else
+      flash[:error] = 'like was not added'
+    end
+    redirect_to user_post_url
+  end
+end
```

Comment:
- [ ] Good job on implementing the add post and comment form however, there isn't any button or icon to like the post, please work on it.

Reply:
Hello @Emmyn5600 are you referring to me or its the two peoples names "@koomekevo" "@Tumworobere" you are reffering to??